Sources/TuistKit/Services/ImportFinderService.swift

```diff
@@ -0,0 +1,105 @@
+import Foundation
+import Path
+import ProjectAutomation
+import Tools
+import TuistCore
+import TuistLoader
+import TuistSupport
+import XcodeGraph
+
+enum ImportFinderServiceError: FatalError, Equatable {
+    case targetNotFound(target: String)
+
+    var description: String {
+        switch self {
+        case let .targetNotFound(target):
+            "Target \(target) was not found"
+        }
+    }
+
+    var type: ErrorType {
+        switch self {
+        case .targetNotFound:
+            .abort
+        }
+    }
+}
+
+final class ImportFinderService {
+    private let manifestGraphLoader: ManifestGraphLoading
+
+    convenience init() {
+        let manifestLoader = ManifestLoaderFactory()
+            .createManifestLoader()
+        let manifestGraphLoader = ManifestGraphLoader(
+            manifestLoader: manifestLoader,
+            workspaceMapper: SequentialWorkspaceMapper(mappers: []),
+            graphMapper: SequentialGraphMapper([])
+        )
+        self.init(
+            manifestGraphLoader: manifestGraphLoader
+        )
+    }
+
+    init(manifestGraphLoader: ManifestGraphLoading) {
+        self.manifestGraphLoader = manifestGraphLoader
+    }
+
+    func run(
+        path: AbsolutePath,
+        targetName: String
+    ) async throws {
+        let (graph, _, _) = try await manifestGraphLoader.load(path: path)
+        let target = graph
+            .projects
+            .map(\.value.targets)
+            .flatMap { $0 }
+            .map(\.value)
+            .filter { $0.name == targetName }
+            .first
+        guard let target else { throw ImportFinderServiceError.targetNotFound(target: targetName) }
+        let imports = Set(await handleTarget(target: target))
+        print("Used imports: \n\(imports.joined(separator: ", "))")
```

Comment:
I'd lint the entire graph, and then output something along the lines of:
```
The following implicit imports have been detected through static code analysis:
  - Target A implicitly imports A, B, and C.
  - Target B implicitly imports X, Y, Z
```

Reply:
To avoid listing system frameworks or libraries we can further filter the results with all known targets from the graph. 

e.g. `UIKit` isn't part of the graph as one of the known targets, and as such would be excluded from the final output.

Reply:
Exactly. You don't want to see the output being noisy with the system frameworks.

Reply:
Now if you launch 
`tuist import_finder --path /Users/rofle100lvl/Desktop/tuist/fixtures/ios_app_with_framework_and_disabled_resources`
You will see

> The following implicit imports have been detected through static code analysis:
> Target App imports: StaticFramework, Framework1

Which is exactly what we want to see